types/kotlin2/NotificationPermissionChanged.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
```

Comment:
Incorrect copyright year in the header. I have a feeling that the template we use is outdated

Reply:
Could you please open a separate issue for that? I agree the template should be updated, but that’s a distinct issue IMO

Reply:
https://github.com/matrix-org/matrix-analytics-events/issues/68